libflux/go/libflux/flux.h

```diff
@@ -75,6 +75,11 @@ void flux_get_env_stdlib(struct flux_buffer_t *);
 // flux_ast_pkg_t* argument, so it should not be reused after calling this function.
 struct flux_error_t *flux_analyze(struct flux_ast_pkg_t *, struct flux_semantic_pkg_t **);
 
+// flux_merge_ast_pkgs merges the files of a given input AST package into the file vector of a
+// given output AST package. This function borrows the packages, but it does not own them. The
+// caller of this function still needs to free the package memory on the Go side.
+struct flux_error_t *flux_merge_ast_pkgs(struct flux_ast_pkg_t *, struct flux_ast_pkg_t *);
```

Comment:
*nit* I would move this prototype closer to the other functions that deal with `flux_ast_pkg_t*` (like parsing) just to keep related functions near to each other.